kubric/core/objects.py

```diff
@@ -29,6 +30,7 @@ class Object3D(base.Asset):
     position (vec3d): the (x, y, z) position of the object.
     quaternion (vec4d): a (W, X, Y, Z) quaternion for describing the rotation.
     up (str): which direction to consider as "up" (required for look_at).
+    front (str): which direction to consider as "front" (required for look_at).
```

Comment:
Don't look_at usually use a "target" point? (from which 1z=normalize(target-position)`)
Front surprises me a little :)

Reply:
Yes, look_at uses a target point. But which side of the object should be pointed towards that target? A: The front. So we need to know which side of the object is considered the front side. If I interpret your parenthesis correctly you implicitly consider -z (?) the front direction (which incidentally is the default value of front)